services/offline/README.md

```diff
@@ -0,0 +1,11 @@
+# DHIS2 App Offline Service
+
+Application configuration for [DHIS2](https://dhis2.org) applications
+
+This library is intended for use with the [DHIS2 Application Platform](https://github.com/dhis2/app-platform).
+
+## Installation
+
+This package is internal to `@dhis2/app-runtime` and generally should not be installed independently.
```

Comment:
(This is not true yet - will be true once released and reexported from the `app-runtime` package)